### PR TITLE
Initial Docs

### PR DESCRIPTION
**Pull request by michaelstingl**
This is a first rough dump of the docs for the new ownCloud iOS app

## Description

+ Antora
+ Navigation
+ Screen
+ Some content (bullet points)

## Related Issue

https://github.com/owncloud/docs/issues/1247

## Motivation and Context

Proper docs are important

## How Has This Been Tested?

if you would like to build a local copy of the iOS documentation, to preview changes that you are making, you can use the following command within the `docs/` directory:

```
yarn install
yarn antora
```

**Previewing the Generated Docs**

Assuming that there are no build errors, the next thing to do is to view the result in your browser. In case you have already installed a web server, you need to configure a virtual host (or similar) which points to the directory `public/`, located in the `docs/` directory of this repository. This directory contains the generated documentation. Alternatively, use the simple server bundled with the current package.json, just execute the following command to serve the documentation at [http://localhost:8080/ios-app/](http://localhost:8080/ios-app/):

```
yarn serve
```

## Screenshots (if appropriate):

iOS_App_Quick_access____ownCloud_iOS_App_Documentation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## TODO

- [ ] Optimise all images
- [ ] Resize images to a standard size
- [ ] Check build process for errors
- [ ] Check/update build process
- [ ] Check formatting
- [ ] Check navigation
- [ ] Check links

### CODE REVIEW

File: docs/modules/ROOT/pages/ios_accounts.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+=== Basic Authentication
+
+* Login Credentials are stored on the device (secure in system keychain).
+* Password manager support.
+
+image:02_basic_auth.png[basic authentication]
+
+=== OAuth 2.0
+
+The OAuth2 implementation uses {sfauthenticationsession-url}[SFAuthenticationSession], which is described as a best practice by {rfc-8252-url}[RFC 8252].
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_accounts.adoc]: @settermjd `doc/SECURITY.md` got updated for iOs 12 in the `feature/sharing-user-groups` branch. (will be merged for initial release.

https://github.com/owncloud/ios-app/blob/feature/sharing-user-groups/doc/SECURITY.md#oauth2-implementation

File: docs/modules/ROOT/nav.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
+* xref:ios_installation.adoc[Getting the ownCloud iOS App]
+
+* Using the iOS App
```
Comment (michaelstingl) [inline docs/modules/ROOT/nav.adoc]: Why not a link?

File: docs/modules/ROOT/pages/ios_accounts.adoc [hunk 3]
```diff
@@ -0,0 +1,91 @@
+= Managing User Accounts
+:toc: right
+:toclevels: 1
+:sfauthenticationsession-url: https://developer.apple.com/documentation/safariservices/sfauthenticationsession
+:rfc-8252-url: https://tools.ietf.org/html/rfc8252#appendix-B.1
+
+The iOS app supports four ways of authenticating user accounts.
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_accounts.adoc]:
```suggestion
The iOS app supports two ways of authenticating user accounts. 
```

File: docs/modules/ROOT/pages/ios_accounts.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+:toclevels: 1
+:sfauthenticationsession-url: https://developer.apple.com/documentation/safariservices/sfauthenticationsession
+:rfc-8252-url: https://tools.ietf.org/html/rfc8252#appendix-B.1
+
+The iOS app supports four ways of authenticating user accounts. 
+These are:
+
+* xref:basic-authentication[Basic Authentication]
+* xref:oauth-2-0[OAuth 2.0]
+* xref:safari-login-session[Safari Login Sessions]
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_accounts.adoc]:
```suggestion
```

File: docs/modules/ROOT/pages/ios_accounts.adoc [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+* xref:the-web-ui[The Web UI]
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_accounts.adoc]:
```suggestion
```

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+* Rename
+* Duplicate
+* Copy
+* Delete
+
+However, file actions are handled slightly differently, depending on whether one or multiple files have been selected.
+You can see the differences in the two images below.
+
+The image below on the left, shows the file popup for actions for _individual_ files. 
+The image below on the right, shows the actions icon row for _multiple-selected_ files.
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_files.adoc]: Second icon is "Move"

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+image:directory-actions.png[]
+
+=== Create Folder
+
+To create a new folder, click btn:[Create folder], enter the name of the new folder, as in the image below, and click btn:[return].
+
+image:create-new-folder.png[]
+
+=== Upload Files
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_files.adoc]: Please make "Upload Files" and "Upload from photo library" to separate sections.

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+== Drag & Drop Files Between Apps (iPad-only)
+
+The iOS app supports the multitasking features on iPad.
+If you open it as a second app with Slide Over, you can use two apps at the same time with Split View and drag and drop one or more files between the two apps.
+Refer to Apple's {multitasking-on-ipad-url}[Multitasking On Your iPad guide] for more information.
+
+.Drag and drop multiple files from ownCloud iOS App to macOS Notes
+image:26_Files_multidragdrop_iPad.png[26_Files_multidragdrop_iPad.png]
+
+=== Upload File From Your Photo Library
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_files.adoc]: Better move this section next to the "Upload Files" section?

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+=== Create Folder
+
+To create a new folder, click btn:[Create folder], enter the name of the new folder, as in the image below, and click btn:[return].
+
+image:create-new-folder.png[]
+
+=== Upload Files
+
+To upload files or any time from your device to your ownCloud server, click btn:[Upload file].
+You will then be able to select one or more recent files, or browse through files from any app that exposes data to the iOS files app.
```
Comment (michaelstingl) [inline docs/modules/ROOT/pages/ios_files.adoc]: I think only upload of individual files possible via iOS FileProvider integration.

File: docs/modules/ROOT/nav.adoc (hunk 2 above)
Comment (settermjd) [reply docs/modules/ROOT/nav.adoc]: Good point. Will add one.

Comment (settermjd) [reply docs/modules/ROOT/nav.adoc]: There is no file that would suit linking to that navigation item, and all of the sub-files are already linked.

File: docs/modules/ROOT/pages/ios_files.adoc (hunk 9 above)
Comment (settermjd) [reply docs/modules/ROOT/pages/ios_files.adoc]: Thank you for the update.

File: docs/modules/ROOT/pages/ios_accounts.adoc (hunk 1 above)
Comment (settermjd) [reply docs/modules/ROOT/pages/ios_accounts.adoc]: Was there a typo in that link, @michaelstingl? I can't find it anywhere.

Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_accounts.adoc]: @settermjd got merged to master:
https://github.com/owncloud/ios-app/blob/master/doc/SECURITY.md#oauth2-implementation

File: docs/modules/ROOT/pages/ios_files.adoc (hunk 9 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_files.adoc]: Ah, found the multiselect path through the fileprovider… 👍

**Comment by michaelstingl** [comment]
@settermjd ready to review from @michaelstingl , @hosy , @mneuwert … ?

**Comment by micbar** [comment]
@settermjd please request some people to review

**Comment by jesmrec** [comment]
Some issues in the content:

1) Using the iOS app > Managing your files > Folder Actions

Screenshot is outdated. 

2) Using the iOS app > Collaboration and links 

Third bullet: `Changing permissions permissions` .  Only one `permissions` is enough ;)

3)  Using the iOS app > Collaboration and links 

As suggestion, i'd set the `Copy Link` section below the `Creation section` . First you create, then you can copy.

4)  Using the iOS app > Settings > Lock Delay

The option `Always` was replaced by `Immediately`. To change in text and screenshot.

5)  Using the iOS app > Settings > Themes

Themes' names got rid of  `ownCloud` word, due to branding regards. To change in text and screenshots

6) Security > Logging

`Mask private Data` was removed from the app. This section can be deleted.

7) Troubleshooting

Logging screenshot is outdated (it seems to be taken before the Log rotation feature)

File: docs/modules/ROOT/pages/ios_collaboration.adoc [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+== Collaborate With Other Users on Files and Folders
+
+image:31_Collab.png[Collaborate With Other Users on Folders and Files]
+
+The Mobile App for iOS supports the following collaboration functionality:
+
+* Adding users and grant permissions
+* Adding groups and grant permissions
+* Changing permissions permissions
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_collaboration.adoc]: only one permissions is enough

File: docs/modules/ROOT/pages/ios_files.adoc [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+
+== Folder Actions
+
+When working with folders, click the plus icon near the top right-hand corner, and three actions become available; these are:
+
+* xref:create-folder[Create folder]
+* xref:upload-files[Upload files]
+* xref:upload-file-from-your-photo-library[Upload file from your photo library]
+
+image:directory-actions.png[Folder actions in ownCloud's iOS App.]
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_files.adoc]: Screenshot outdated

File: docs/modules/ROOT/pages/ios_collaboration.adoc [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+Just as with the web interface, the iOS app lets you grant access to both files and folders with links.
+Specifically, you can do the following:
+
+* xref:copy-public-links[Copy public links]
+* xref:create-private-links[Create private links]
+* xref:set-passwords-and-expiration-dates[Set passwords and expiration dates]
+* xref:send-private-link-urls-to-other-apps[Send private link URLs to other apps (mail, messenger)]
+* xref:delete-private-links[Delete private links]
+
+=== Copy Public Links
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_collaboration.adoc]: As suggestion, i'd set the `Copy Link` section below the `Creation section` . First you create, then you can copy

File: docs/modules/ROOT/pages/ios_settings.adoc [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+
+To protect access to the iOS app with a Passcode, enable btn:[Passcode Lock] in the Setting's Security section.
+You will then be prompted to enter, and repeat, a 4-digit Passcode.
+
+image:settings-security-passcode-enabled.png[Enable a Passcode lock in ownCloud's iOS App for iPhone and iPad.]
+
+=== Lock Delay
+
+When a Passcode is enabled, the app will be locked every time you change to another application.
+However, under menu:Settings[Security > Lock application], you can choose to only lock the application after 1, 5, or 30 minutes, instead of "_always_", which is the default.
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_settings.adoc]: The option `Always` was replaced by `Immediately`.

File: docs/modules/ROOT/pages/ios_settings.adoc [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+
+image:settings-lock-application-duration.png[Set the application lock duration in ownCloud's iOS App for iPhone and iPad.]
+
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_settings.adoc]: Screenshot outdated. The option `Always` was replaced by `Immediately`.

File: docs/modules/ROOT/pages/ios_settings.adoc [hunk 15]
```diff
@@ -0,0 +1,97 @@
...
+To revoke one or more previously approved certificates, first navigate to menu:Settings[Certificates] (for any one of your registered accounts).
+Then, in the "_User-Approved Certificates_" section, swipe left on the certificate(s) that you wish to revoke and press btn:[Revoke approval].
+
+== Themes
+
+The iOS app comes with three themes: 
+
+* ownCloud Light
+* ownCloud Dark; and
+* ownCloud Classic
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_settings.adoc]: Themes' names got rid of  `ownCloud` word, due to branding regards. Now they are `Light`, `Dark` and `Classic`.

File: docs/modules/ROOT/pages/ios_settings.adoc [hunk 16]
```diff
@@ -0,0 +1,97 @@
...
+
+To change the theme, navigate to menu:Settings[Theme], and pick the one that you want.
+
+image:84_Settings_themes.png[84_Settings_themes.png]
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_settings.adoc]: Screenshot outdated. Themes' names got rid of ownCloud word, due to branding regards. Now they are Light, Dark and Classic.

File: docs/modules/ROOT/pages/ios_security.adoc [hunk 17]
```diff
@@ -0,0 +1,163 @@
...
+The build script that created the {ios-sdk-openssl-lib-url}[OpenSSL binaries] used in the app is available in the SDK's {ios-sdk-openssl-build-script-url}[GitHub repository] and can be used to reproduce the build result.
+
+NOTE: OpenSSL is used solely to provide detailed summaries of SSL/TLS certificates - functionality that iOS is currently missing.
+
+=== Logging
+
+When logging information, parts of the log message can be tagged as private. 
+If "*Mask private data*" is enabled, under menu:Settings[Logging] (it is by default), these parts will be replaced with `«private»` before the log message is written.
+
+image:masking-private-data.png[]
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_security.adoc]: `Mask private Data` was removed from the app. This section can be deleted.

File: docs/modules/ROOT/pages/ios_troubleshooting.adoc [hunk 18]
```diff
@@ -0,0 +1,110 @@
...
+** Log file.
+
+Once these have been enabled:
+
+* Click btn:[Reset log file].
+* Perform the steps to reproduce the error
+* Go back to the Logging settings and click btn:[Share log file].
+
+.All iOS App logging settings enabled and set
+image:ios-app-settings-logging.png[ios-app-settings-logging]
```
Comment (jesmrec) [inline docs/modules/ROOT/pages/ios_troubleshooting.adoc]: Screenshot outdated. It seems to be taken before the Log rotation feature

File: docs/modules/ROOT/pages/ios_security.adoc (hunk 17 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_security.adoc]: Solved with 9e221ab6d78890997cb59a5a6f57f2f4e43ccb0b

File: docs/modules/ROOT/pages/ios_settings.adoc (hunk 15 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_settings.adoc]: Solved with 6f7664a70be9091d2309da76c100d10d79bffc6f

File: docs/modules/ROOT/pages/ios_troubleshooting.adoc (hunk 18 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_troubleshooting.adoc]: Won't fix for now – screenshots need to be updated anyway

File: docs/modules/ROOT/pages/ios_settings.adoc (hunk 16 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_settings.adoc]: Won't fix for now – screenshots need to be updated anyway

File: docs/modules/ROOT/pages/ios_settings.adoc (hunk 14 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_settings.adoc]: Won't fix for now – screenshots need to be updated anyway

File: docs/modules/ROOT/pages/ios_files.adoc (hunk 11 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_files.adoc]: Won't fix for now – screenshots need to be updated anyway

File: docs/modules/ROOT/pages/ios_collaboration.adoc (hunk 12 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_collaboration.adoc]: Should be resolved with 4ea032d546836cef88ce32fe06ed3484b4f5d068

File: docs/modules/ROOT/pages/ios_accounts.adoc (hunk 1 above)
Comment (michaelstingl) [reply docs/modules/ROOT/pages/ios_accounts.adoc]: Improvement in 51de1bafc55eca5f9ebe7d3e7fcf5381af0bd376

**Pull request merged**